docs/docs/webchat/webchat-storage.md

```diff
@@ -0,0 +1,49 @@
+---
+id: webchat-storage
+title: Storage
+---
+
+## Local and Session Storage
+
+By default, Botonic stores its state in the [**localStorage**](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) browser where the data doesn't expire.
+
+However, for security and privacy reasons, you may have to avoid storing bot data.
```

Comment:
However, for security and privacy reasons, you may have to avoid storing bot data when the page session ends.